File: java_files/pdp_planet_travel_sim/src/app/Simulation.java
/**
*
*@author Sude Andurman	sude.andurman@ogr.sakarya.edu.tr
*@since 04.04.2025
*<p>
*	Bilgisayar Mühendisliği 2. Sınıf 
*	Programlama Dilleri Prensipleri 1/C şubesi 
*</p>
*/


package app;

public class Simulation {

	public static void main(String[] args)
	{
		/* MAKE AN OBJECT FROM FACADE */
		StartSim_Facade start;
		start = new StartSim_Facade();
		
		/* START THE SIM */
		start.runSim();
	}

}

File: c_files/src/Simulation.c
/**
*
*@author Sude Andurman	sude.andurman@ogr.sakarya.edu.tr
*@since 14.05.2025
*<p>
*	Bilgisayar Mühendisliği 2. Sınıf 
*	Programlama Dilleri Prensipleri 1/C şubesi 
*</p>
*/

#include "StartSim_Facade.h"

int main()
{
	StartSim_Facade sim = new_StartSim_Facade();
	if (sim == NULL)
	{
		printf("Memory allocation failed at main\n");
		return (1);
	}
	sim->runSim(sim);
	sim->delete_StartSim_Facade(sim);

	return (0);
}